### PR TITLE
Add Notifications Component

### LINKED ISSUE #2603: Skeleton component for Notifications
## ✨ What new feature do you want to see?

Create card component for notifications that will show notifications when:
* There is an authorization that is expiring in 30 days or less.

_Preliminary design_

image

## 🇺🇸 States this change applies to
- [X] ALL
- [ ] Nebraska
- [ ] Illinois

### PR DESCRIPTION
**Pull request by abdulfataiaka**
## 💅🏼 What issue does this fix?

closes #2603

* Add skeleton component for notification messages

## 🍂 Before You Submit

* [ ] Did you write & run tests and linters?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Migrations
* [ ] Dependencies

### CODE REVIEW

**Comment by abdulfataiaka** [comment]
Where to put the styles?
Is the short divider necessary?
Expected date format from server

**Comment by cjhaddad** [comment]
> Where to put the styles? Is the short divider necessary? Expected date format from server

Hello! A few comments for this PR: 
-- for the styling can you use tailwind css classes instead? You then shouldn't need to worry about where to put the styles.
-- can you relocate the 'Notifications' component and test files to the Dashboards directory and delete the 'Notifications' directory? The notifications component will be used on the Dashboard view. 
-- Don't worry about adding the Notifications component to the Dashboard table, yet. This can come with the ticket that does the styling changes to the Dashboard summaries section. That said, can you remove it from the AttendanceView and restore this file? 

Thank you so much @abdulfataiaka otherwise this all looks great!

File: client/src/AttendanceView/AttendanceView.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { ReactComponent as EditIcon } from '../_assets/editIcon.svg'
 import '_assets/styles/edit-icon.css'
```
Comment (cjhaddad) [inline client/src/AttendanceView/AttendanceView.js:11]: restore file and remove Notifications

File: client/src/Notifications/Notifications.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Collapse, Divider } from 'antd'
+import { ExclamationCircleOutlined } from '@ant-design/icons'
+
+const { Panel } = Collapse
+
+const styles = {
```
Comment (cjhaddad) [inline client/src/Notifications/Notifications.js:8]: use Tailwind css classes instead

File: client/src/Notifications/Notifications.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
```
Comment (cjhaddad) [inline client/src/Notifications/Notifications.js:1]: relocate this component file and the test file to the 'Dashboard' directory

**Review by cjhaddad — CHANGES_REQUESTED** [review]

**Review by cjhaddad — APPROVED** [review]
Looks great! Thank you @abdulfataiaka !

**Comment by cjhaddad** [comment]
@abdulfataiaka just rebase main and it'll be good to go.

**Review by katelovescode — DISMISSED** [review]

**Review by katelovescode — APPROVED** [review]

**Pull request merged**